### ISSUE TITLE
Accented characters mess up the rest of the text

### ISSUE THREAD

**Issue by Bob74** [Guad/NativeUI#74]
When using accented characters in item text or description, the in game result is very buggy:
![nativeui](https://user-images.githubusercontent.com/9498543/36285186-d68b5cf4-12aa-11e8-80a2-2b5e10906352.png)
What was supposed to be written was:
```
ÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂÂ.A B C D E F G H I J K L M N O P Q R S T U V W X Y Z 0 1 2 3 4 5 6 7 8 9 A B C D E F G H I J K L M N O P Q R S T U V W X Y Z 0 1 2 3 4 5 6 7 8 9
```
As you can see on the first line, the text doesn't automatically creates a new line and there are missing characters in the text:
M N O P Q R 7 8 9
M N O P Q R **S T U V W X Y Z 0 1 2 3 4 5 6** 7 8 9

It is very annoying when you need to write something in a language using accented characters...
I saw the same kind of problem with Russian characters, so maybe the problem comes from NativeUI only supporting simple ASCII characters?

At the moment, I put spaces in my text where the letters are "eaten" but it is ugly af.

**Comment by justalemon** [comment]
@Guad Do you know if this is related to #71?

**Comment by Guad** [comment]
@justalemon absolutely. Using the 0x9040DFB09BE75706 native would fix all of this ideally. IIRC fivem fork does this?

**Comment by justalemon** [comment]
This has been fixed, see #71 for more information.
